Library/Formula/camlp5.rb

```diff
@@ -13,20 +13,33 @@ class Camlp5 < Formula
 
   deprecated_option "strict" => "with-strict"
   option "with-strict", "Compile in strict mode"
+  option "with-doc", "Install the html and info documentation"
+  option "with-latex", "Install the pdf, ps, and tex documentation"
```

Comment:
> The reason I did that is so that I could add a comment about what it does. Can a comment be added to an => :optional?

Yup. When you add an `optional` dependency we automatically generate text for it, but if it has an explicit option that text is used instead. So you can use `option "with-tex"` and keep the custom comment.